gcsweb.k8s.io/deployment.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: gcsweb
+  namespace: gcsweb
```

Comment:
Historically I left the NS out of the YAML so that we could do canaries to another namespace (as in k8s.io). 

Obviously this is a tradeoff, and I am happy to discuss :)

Reply:
We can override this at deploy time. I personally like the idea of not deploying to default if someone forgets

Reply:
If the YAML and the `-n` disagree it fails (last I tried).  I'm happy to cross that bridge later